AIPscan/models.py

```diff
@@ -398,7 +399,7 @@ class File(db.Model):
     format_version = db.Column(db.String(255))
     checksum_type = db.Column(db.String(255))
     checksum_value = db.Column(db.String(255), index=True)
-    premis_object = db.Column(db.Text())
+    premis_object = db.Column(LONGTEXT)
```

Comment:
I think this is the reason [tests are currently failing](https://github.com/artefactual-labs/AIPscan/actions/runs/17618665997/job/50058456352) and this needs to support `sqlite` as well?